assets/i18n/qqq.json

```diff
@@ -226,6 +233,126 @@
 	"I18N_ERROR_PAGE_TITLE_401": "Title of the page displayed when a 401 type error occurred because the user has no permissions to access the page.",
 	"I18N_ERROR_PAGE_TITLE_404": "Title of the page displayed when a 404 type error occurred because the page was not found.",
 	"I18N_ERROR_PAGE_TITLE_500": "Title of the page displayed when a 500 type error occurred because the server failed to process the request.",
+	"I18N_EXPLORATION_-tMgcP1i_4au_DESCRIPTION": "Description of the chapters displayed on story viewer page.",
+    "I18N_EXPLORATION_-tMgcP1i_4au_TITLE": "Title of the chapters displayed on topic viewer and story viewer page.",
```

Comment:
The indentation for these lines seems different from the lines above (if you look at the Files Changed tab).

Reply:
Yes, there is some problem in my local indentation thing, it does not show this spacing issue until I remove the indentation and press tab again. Fixed this!